src/app/core/services/upload.service.ts

```diff
@@ -3,9 +3,18 @@ import { GithubService } from './github.service';
 import { uuid } from '../../shared/lib/uuid';
 import { throwError } from 'rxjs';
 
-export const SUPPORTED_FILE_TYPES = ['gif', 'jpeg', 'jpg', 'png', 'docx', 'gz', 'log', 'pdf', 'pptx', 'txt', 'xlsx', 'zip'];
+const SUPPORTED_VIDEO_FILE_TYPES = ['mp4', 'mov'];
+export const SUPPORTED_FILE_TYPES = ['gif', 'jpeg', 'jpg', 'png', 'docx', 'gz', 'log', 'pdf', 'pptx', 'txt', 'xlsx', 'zip']
+                                    .concat(SUPPORTED_VIDEO_FILE_TYPES);
```

Comment:
Its good that we separate the video file types to make it easier to test later! Would joining the array this way be cleaner perhaps?

```suggestion
export const SUPPORTED_FILE_TYPES = ['gif', 'jpeg', 'jpg', 'png', 'docx', 'gz', 'log', 'pdf', 'pptx', 'txt', 'xlsx', 'zip', ...SUPPORTED_VIDEO_FILE_TYPES];
```

Reply:
Good idea!